modules/about-etcd-encryption.adoc

```diff
@@ -16,4 +16,9 @@ When you enable etcd encryption, the following OpenShift API server and Kubernet
 * OAuth access tokens
 * OAuth authorize tokens
 
-When you enable etcd encryption, encryption keys are created. These keys are rotated on a weekly basis. You must have these keys in order to restore from an etcd backup.
+When you enable etcd encryption, encryption keys are created. These keys are rotated on a weekly basis. You must have these keys to restore from an etcd backup.
```

Comment:
So since your original PR didn't actually touch anything on this line, this cherry pick shouldn't either. You'd want to keep the original wording that was in 4.6 and not introduce another change (of dropping the "in order"). 

Do you mind going back and adding the "in order" back in?